src/bun.js/bindings/bindings.zig

```diff
@@ -2927,6 +2927,17 @@ pub const JSGlobalObject = opaque {
         this.vm().throwError(this, err);
     }
 
+    pub fn throwMissingArgumentsValue(this: *JSGlobalObject, arg_names: []const []const u8) JSValue {
```

Comment:
I think `arg_names` should be comptime so we don't accidentally panic at runtime

Reply:
was worried about about generic instantiation bloat the function is so small i can make it `inline` too. the call it makes is non-inline so it wont explode by doing so